common/src/main/java/com/wynntils/models/map/PoiModel.java

```diff
@@ -78,11 +82,26 @@ private void loadData() {
         // These are loaded serially after places instad
         if (WynntilsMod.isDevelopmentEnvironment()) return;
 
+        loadCaves();
         loadServices();
         loadCombat();
         loadCustomPoiProviders();
     }
 
+    private void loadCaves() {
+        Download dl = Managers.Net.download(UrlId.DATA_STATIC_CAVE_INFO);
+        dl.handleReader(reader -> {
+            Type type = new TypeToken<List<CaveProfile>>() {}.getType();
+
+            List<CaveProfile> profiles = WynntilsMod.GSON.fromJson(reader, type);
+
+            cavePois.addAll(profiles.stream()
+                    .map(profile ->
+                            new CombatPoi(PoiLocation.fromLocation(profile.location), profile.name, CombatKind.CAVES))
+                    .collect(Collectors.toUnmodifiableSet()));
+        });
+    }
```

Comment:
Ended up loading this here since otherwise it would be really messy..

Reply:
I think that seems fine. Not sure what you should have done otherwise...